From 275fe9da663193a843de450f03e810daedc06955 Mon Sep 17 00:00:00 2001
From: Aaron Conole <aconole@redhat.com>
Date: Mon, 3 Aug 2020 15:25:38 -0400
Subject: [PATCH] lldp_8021qaz: extern config object

The config object only exists as part of the config translation unit
so remove the extra config object in the 8021qaz module.

Signed-off-by: Aaron Conole <aconole@redhat.com>
[Retrieved from:
https://github.com/intel/openlldp/pull/55/commits/275fe9da663193a843de450f03e810daedc06955]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 lldp_8021qaz.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lldp_8021qaz.c b/lldp_8021qaz.c
index a42fd11..673d758 100644
--- a/lldp_8021qaz.c
+++ b/lldp_8021qaz.c
@@ -49,7 +49,7 @@
 
 
 struct lldp_head lldp_head;
-struct config_t lldpad_cfg;
+extern config_t lldpad_cfg;
 extern bool read_only_8021qaz;
 
 static int ieee8021qaz_check_pending(struct port *port, struct lldp_agent *);
